README.rst

```diff
@@ -3,53 +3,38 @@ GraphAr
 
 |GraphAr CI| |Docs CI| |GraphAr Docs|
 
-GraphAr (short for "Graph Archive") is an open source, standard data file format with C++ SDK and Spark tools for graph data storage and retrieval.
-
-The GraphAr project includes such modules as:
-
-- The design of the standardized file format (GAR) for graph data.
-- A C++ Library for reading and writing GAR files.
-- Apache Spark tools for generating, loading and transforming GAR files (coming soon).
-- Examples of applying GraphAr to graph processing applications or existing systems such as GraphScope.
-
-
+Welcome to GraphAr (short for "Graph Archive"), an open source, standardized file format for graph data storage and retrieval.
 
+What is GraphAr?
+-----------------
 
 |Overview Pic|
 
+Graph processing serves as the essential building block for a diverse variety of
+real-world applications such as social network analytics, data mining, network routing,
+and scientific computing.
 
-Motivation
-----------
-
-Graph processing serves as the essential building block for a diverse variety of real-world applications such as social network analytics, data mining, network routing, and scientific computing.
-
-GraphAr (GAR) is established to enable diverse graph applications and systems (in-memory and out-of-core storages, databases, graph computing systems and interactive graph query frameworks) to build and access the graph data conveniently and efficiently. It specifies a standardized system-independent file format for graph and provides a set of interfaces to generate and access such formatted files.
-
-GraphAr (GAR) targets two main scenarios:
-
-- To serve as the standard file format for importing/exporting and persistent storage of the graph data for diverse existing systems, reducing the overhead when various systems co-work.
-- To serve as the direct data source for graph processing applications.
-
-
-What's in GraphAr
----------------------
+GraphAr is a project that aims to make it easier for diverse applications and
+systems (in-memory and out-of-core storages, databases, graph computing systems, and interactive graph query frameworks)
+to build and access graph data conveniently and efficiently.
 
-The **GAR** file format that defines a standard store file format for graph data.
+To achieve this, GraphAr provides:
 
-The **GAR SDK** library that contains a C++ library to provide APIs for accessing and generating the GAR format files.
+- The Graph Archive(GAR) file format: a standardized system-independent file format for storing graph data
+- Libraries: a set of language-independent libraries for reading and writing GAR files
```

Comment:
Why use "language-independent" here? And it is a little confusing that "Libraries" do not contain the Spark library.